pingen/__init__.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
 #
-#    Author: Guewen Baconnier
-#    Copyright 2012 Camptocamp SA
+#    Author: Anar Baghirli
+#    Copyright 2017 Camptocamp SA
```

Comment:
1. Don't change the author, you are not the original author of that module.
You can add your name as contributor in the README.rst
It's probably not what you meant to do but this is like stealing the work of someone else.
Add only your name as author if you are adding a functionnality or a major work in the code.

2. Dates shouldn't be erased, it should list the years of edition in this case 2012, 2017

3. Use the following format for headers:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3